### DISCUSSION TITLE
Data Base integrity Check  User script makes temp db grow

### DISCUSSION THREAD

**Discussion by nakatsue** [olahallengren/sql-server-maintenance-solution#816] [Questions & Answers]
Hello, I have a problem executing the script. It makes my tempdb grow. That is normal?? Is there any parameter to avoid that?

**Comment by griffitmatt** [comment]
Hi it's most likely the integrity check command being used by the ola script not the ola script itself thats causing the tempdb to grow. Refer to this article for more info. https://www.sqlskills.com/blogs/paul/how-does-dbcc-checkdb-with-estimateonly-work/

**Reply by nakatsue** [reply]
I have read the article but I see that it is based on a bug from SQL Server 2008 to 2012. I am using SQL Server 2017

**Reply by griffitmatt** [reply]
Apologies it was more about highlighting the dbcc checkdb command will consume tempdb space. There is no way around this so you will need to ensure there is adequate space available for the tempdb.

**Comment by Stiffboard** [comment]
You need to read the documentation on dbcc @ hallengren.com. there are also links to all dbcc commands @ microsoft.com. It's perfectly normal that tempdb is used.
